backend/base/views/user_views.py

```diff
@@ -122,6 +125,9 @@ def updateUser(request, pk):
     user.email = data["email"]
```

Comment:
* ten update powinien być realizowany przez serializer
* każdy kto ma konto będzie mógł zmieniać innym dane

Reply:
przeniosłem ten widok do klasy z permission_classes jako IsAdminUser oraz update jest już teraz realizowany przez serializer, ale działa to tylko w połowie. Oryginalne pola modelu email i username aktualizują się poprawnie, natomiast te nieoryginalne - name i isAdmin - nie atualizują się. Metody get_name i get_isAdmin są zdefiniowane, dlatego nie wiem dlaczego to nie działa.

Reply:
Jeśli dane są zwracane przez metody to są one read_only. Najlepiej zrobić osobny serializer do update i tam to obsłużyć